src/Sylius/Behat/Context/Ui/Shop/CartContext.php

```diff
@@ -31,6 +33,8 @@ final class CartContext implements Context
     public function __construct(
         private SharedStorageInterface $sharedStorage,
         private SummaryPageInterface $summaryPage,
+        private AddressPageInterface $addressPage,
+        private CheckoutSubtotalElementInterface $checkoutSubtotalElement,
```

Comment:
It is not a BC break, but this change could be mentioned in the UPGRADE file in Behat section
And I will be for moving `$checkoutSubtotalElement` after other element `@cartWidgetElement `

Reply:
What exactly would you like to have in the UPGRADE file 🤔?

Reply:
I thought about a simple note with information about adding new arguments to the constructor, because it can break tests in the end application if someone overwrites this context but we probably haven't done that until now 🤔

Reply:
I have mixed feelings about it 🤔. I've never spotted a note like this before. We can leave this conversation open for further feedback from other guys.